doc/api/v8.md

```diff
@@ -159,6 +159,59 @@ v8.setFlagsFromString('--trace_gc');
 setTimeout(() => { v8.setFlagsFromString('--notrace_gc'); }, 60e3);
 ```
 
+## v8.triggerHeapdump([filename])
```

Comment:
I suspect there will be future feature requests about returning this as a string or an object instead of directly writing it to a file, can this API be something like `v8.triggerHeapdump().write()`, where `v8.triggerHeapdump()` can return a wrapper around `v8::HeapSnapshot`? Then we can just overwrite its `toString` and `toJSON` methods and implement `Stream.Readable` on top of it in the future.

Reply:
If that's needed, I'd prefer to just to bring back a version of `getHeapdump()` that that returns a stream.Readable and keep `triggerHeapdump()` as it is.